src/Pmi/Controller/ReviewController.php

```diff
@@ -30,19 +33,52 @@ public function todayAction(Application $app, Request $request)
             return $app->redirectToRoute('home');
         }
 
+        $review = new Review($app['db']);
+
         // Get beginning of today (at midnight) in user's timezone
-        $startString = 'today';
-        // Allow overriding start time to test in non-prod environments
-        if (!$app->isProd() && intval($request->query->get('days')) > 0) {
-            $startString = '-' . intval($request->query->get('days')) . ' days';
+        $startDate = new \DateTime('today', new \DateTimeZone($app->getUserTimezone()));
+
+        // Get end of today (at midnight) in user's timezone
+        $endDate = new \DateTime('yesterday 1 sec ago', new \DateTimeZone($app->getUserTimezone()));
```

Comment:
I think we can simplify a lot of the date math by:

1. Setting the form's view timezone to the local timezone
2. Changing the query to use `< :endTime` instead of `<=` which allows us to just use midnight of the following day instead of `23:59:59` of the specified day

See diff from this branch:
https://github.com/all-of-us/healthpro/compare/sn/hpro-658/new-todays-participants...jt/HPRO-658/todays-participants-simplify-date-math

Reply:
I also made some minor text changes to the error message and display message.